src/refda/exec_item.h

```diff
@@ -30,6 +31,11 @@ extern "C" {
 // Forward declaration
 typedef struct refda_exec_seq_s refda_exec_seq_t;
 
+#define REFDA_EXEC_PENDING  0
```

Comment:
Can these be made as a C enum for consistency? Then you don't need to assign explicit integer values.
The atomic state can still be `int` as all C enums are typed as int values.

Reply:
No problem, these have been refactored into an `enum`.